File: socialgo/src/main/java/com/fungo/socialgo/share/listener/OnShareListener.java
package com.fungo.socialgo.share.listener;


import com.fungo.socialgo.share.config.PlatformType;

/**
 * 分享回调监听
 * Created by tsy on 16/8/5.
 */
public interface OnShareListener {
    void onComplete(PlatformType platform_type);

    void onError(PlatformType platform_type, String err_msg);

    void onCancel(PlatformType platform_type);
}

File: socialgo/src/main/java/com/fungo/socialgo/share/listener/OnShareListener.kt
package com.fungo.socialgo.share.listener

import com.fungo.socialgo.share.config.PlatformType

/**
 * @author Pinger
 * @since 18-7-20 上午9:56
 * 分享回调
 */

interface OnShareListener {

    /**
     * 分享完成
     */
    fun onComplete(platform_type: PlatformType)

    /**
     * 分享完成
     */
    fun onError(platform_type: PlatformType, err_msg: String)

    /**
     * 分享完成
     */
    fun onCancel(platform_type: PlatformType)

}